# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/angle/checkout/src/libANGLE/renderer/d3d/d3d11/Image11.h
# Commit: 13282d7a47a5
# Full Hash: 13282d7a47a57a73be8d060efd9645b4d642f03e
# Author: Miko Mynttinen <mikokm@gmail.com>
# Date: 2019-09-08 09:46:52
# Regressor Bug: 1578576
# File Overlap Count: 4
# Description:
#   Bug 1578576 - Part 2: Update to ANGLE 3865 r=jgilbert
#   
#   Depends on D44561
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D44579
# ==============================================================================

diff -r 3064469c073d -r 13282d7a47a5 gfx/angle/checkout/src/libANGLE/renderer/d3d/d3d11/Image11.h
--- a/gfx/angle/checkout/src/libANGLE/renderer/d3d/d3d11/Image11.h	Sat Sep 07 13:36:46 2019 +0000
+++ b/gfx/angle/checkout/src/libANGLE/renderer/d3d/d3d11/Image11.h	Sat Sep 07 13:38:36 2019 +0000
@@ -13,6 +13,7 @@
 #include "common/debug.h"
 #include "libANGLE/ImageIndex.h"
 #include "libANGLE/renderer/d3d/ImageD3D.h"
+#include "libANGLE/renderer/d3d/d3d11/MappedSubresourceVerifier11.h"
 #include "libANGLE/renderer/d3d/d3d11/renderer11_utils.h"
 
 namespace gl
@@ -113,6 +114,7 @@
     DXGI_FORMAT mDXGIFormat;
     TextureHelper11 mStagingTexture;
     unsigned int mStagingSubresource;
+    MappedSubresourceVerifier11 mStagingTextureSubresourceVerifier;
 
     bool mRecoverFromStorage;
     TextureStorage11 *mAssociatedStorage;